binding/Binding/SKTypeface.cs

```diff
@@ -108,6 +110,46 @@ public int CharsToGlyphs (IntPtr str, int strlen, SKEncoding encoding, out ushor
 				}
 			}
 		}
-	}
+
+        public string FamilyName {
+            get {
+                var r = SkiaApi.sk_string_new_empty();
```

Comment:
@agallero - this might cause GC issues as this type may be moved in memory while we are in unmanaged code. To avoid this, we have get a GC handle... Might be better to move this whole operation into unmanaged code and allocate there. Shouldn't be a problem as we are going to be returning that result (SKString). The GC can then clean that up when not in use anymore